assets/javascripts/hours_tracker.js

```diff
@@ -78,7 +78,7 @@ export default class HoursTracker {
     }
 
     setTitle(time) {
-        const components = [window.RedmineTracky.documentTitle, this.workDescription(), time];
+        const components = [window.RedmineTracky.documentTitle, this.workDescription()];
```

Comment:
Proposal for the document title:
`"#{Project}" - #{time_without_seconds}` e.g: "one11 - 0:30"

* I don't want to see "tracky" (event though it's a cool plugin)
* I always write my description at the and and they are rather long anyways.

Reply:
🤔 I'll look into it ^^ getting the project is a bit tricky